src/xrCore/Animation/Motion.hpp

```diff
@@ -40,7 +40,7 @@ struct st_BoneMotion
         m_Flags.zero();
         ZeroMemory(envs, sizeof(CEnvelope*) * ctMaxChannel);
     }
-    void SetName(LPCSTR nm) { name = nm; }
+    void SetName(pcstr nm) { name = nm; }
```

Comment:
Why you have 2 commits named 'xrCore: refactor - replace LPCSTR to pcstr ' in the history - 5db1f57 and db10f95?

Reply:
Watch carefully, one LPCSTR, another LPSTR